lib/gen_mqtt.ex

```diff
@@ -19,7 +19,7 @@ defmodule GenMQTT do
         end
 
         def on_publish(["room", location, "temp"], message, state) do
-          IO.puts "It is #\{message\} degrees in #\{location\}"
```

Comment:
I am not sure this escaping is unnecessary; does the test suite still run without them?